action.yml

```diff
@@ -6,7 +6,7 @@ branding:
   color: green
 inputs:
   github_token:
-    description: 'Token for the repo. Can be passed in using $\{{ secrets.GITHUB_TOKEN }}'
+    description: 'GitHub Organization Token for the repo to be synced from. Can be passed in using $\{{ secrets.GITHUB_TOKEN }}'
```

Comment:
Think `token` is somehow more general than `GitHub Organization` Token because the token can e.g. also be a PAT